lib/new_relic/transaction/event.ex

```diff
@@ -23,7 +24,8 @@ defmodule NewRelic.Transaction.Event do
         timestamp: transaction.timestamp,
         name: transaction.name,
         duration: transaction.duration,
-        type: transaction.type
+        type: transaction.type,
+        queueDuration: transaction.queue_duration
```

Comment:
This isn't where we should be reporting the attribute, only a small set of attributes are accepted here, we can remove this from `Event`.